frontend/src/helpers/datetime.ts

```diff
@@ -22,3 +22,85 @@ export const formatDateTime = ({
   }
   return format(date, dateFormat);
 };
+
+// Helper function to convert duration to seconds
+export const durationToSeconds = (
```

Comment:
Not sure the purpose of this code when the ms library already does this in both direction

https://github.com/vercel/ms?tab=readme-ov-file#convert-from-milliseconds